tests/ab/testServer.php

```diff
@@ -16,7 +16,7 @@
     });
 });
 
-$server = new Server([$ws]);
+$server = new Server($loop, $ws);
```

Comment:
@WyriHaximus This use of `WebsocketMiddleware` causes an exception because `__invoke` requires the `$next` argument. This is not a new bug, but what are your thoughts on making `$next` optional on `__invoke` so it can be last in the handler list?

Reply:
Like the commit I just pushed?